packages/transaction-controller/CHANGELOG.md

```diff
@@ -6,6 +6,33 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [9.0.0]
+### Added
+- **BREAKING**: Add required `getSelectedAddress` callback argument to constructor ([#1579](https://github.com/MetaMask/core/pull/1579))
```

Comment:
Nit: I can see how it makes sense to phrase this change entry as "Added required ___", but it feels more appropriate under the "Changed" categories. Added is more often used for "new features" that can be taken advantage of, whereas this is a change in expectations for the caller. Likewise for the second entry as well.

Reply:
Great point, done.